Makefile

```diff
@@ -167,7 +168,9 @@ RUSTC_SRC_DL := $(RUSTCSRC)/dl-version
 MAKE_MINICARGO = $(MAKE) -f minicargo.mk RUSTC_VERSION=$(RUSTC_VERSION) RUSTC_CHANNEL=$(RUSTC_SRC_TY) OUTDIR_SUF=$(OUTDIR_SUF)
 
 
-output$(OUTDIR_SUF)/libstd.rlib: $(RUSTC_SRC_DL) $(BIN)
+$(MINICARGO): $(RUSTC_SRC_DL) $(MRUSTC)
```

Comment:
minicargo doesn't depend on these - It has its own source directory.

If you're going this far, might be an idea to move all of the "build rust code" logic to `minicargo.mk`